docs/source/en/features/distributed_adafactor.md

```diff
@@ -0,0 +1,114 @@
+# Distributed Adafactor
+
+Author:
+
+**Related Paper**
+- [Adafactor: Adaptive Learning Rates with Sublinear Memory Cost](https://arxiv.org/abs/1804.04235)
+
+## Introduction
+
+Distributed Adafactor is an optimiser that supports hybrid optimisation, including 1D tensor parallelism as well as ZerO. It makes full use of computational resources through reasonable task parallelism, improves training efficiency and speed, and reduces space pressure on single card storage. It has a wide range of applications and currently supports a range of Transformer based models, see [tests.kit.model_zoo](https://github.com/hpcaitech/ColossalAI/tree/main/tests/kit/model_zoo) for details.
+
+### API Reference
+
+{{ autodoc:colossalai.nn.optimizer.distributed_adafactor.DistributedAdaFactor }}
+
+## Hands-On Practice
+We now demonstrate how to start Distributed Adafactor with booster API.
+### step 1. Import libraries
+
+```python
+import torch
+from torch import nn
+import torch.distributed as dist
+from transformers import LlamaModel, LlamaConfig
+
+from colossalai.cluster import ProcessGroupMesh
+from colossalai.shardformer.layer import Linear1D_Col, Linear1D_Row
+from colossalai.nn.optimizer.distributed_adafactor import DistributedAdaFactor
+from colossal_llama2.dataset.loader import load_tokenized_dataset
+from colossalai.booster.plugin import GeminiPlugin, HybridParallelPlugin, LowLevelZeroPlugin
+```
+
+### step 2. Initialize Distributed Environment and Parallism Group
+We then need to initialize distributed environment. For demo purpose, we uses `colossalai.launch`. You can refer to [Launch Colossal-AI](../basics/launch_colossalai.md)
+for other initialization methods. We use `ProcessGroupMesh` to create tensor parallelism group and data parallelism group.
+
+```python
+# Distributed Enviroment
+config = {}
+colossalai.launch(config=config, rank=rank, world_size=world_size,host="localhost", port=port, backend="nccl")
+
+# Parallism Group
+tp_size, zero_size = 2, 2
+use_zero = True if zero_size > 1 else False
+proc_mesh = ProcessGroupMesh(tp_size, zero_size)
+tp_group, dp_group = proc_mesh.get_group_along_axis(0), proc_mesh.get_group_along_axis(1)
```

Comment:
These are not needed

Reply:
Removed in https://github.com/hpcaitech/ColossalAI/pull/5484/commits/1038b233632c5ad04c40262a7027ba7205a14e65;